projects/demo/src/modules/components/button/index.html

```diff
@@ -23,7 +23,7 @@
                 [loading]="loading"
                 [size]="size"
             >
-                Button
+                TODO: revert this line before merge
```

Comment:
```suggestion
                Button
```

(apply suggestion before merge)

Reply:
I've checked everything – it works (no duplications)

<img width="1022" alt="report" src="https://github.com/taiga-family/taiga-ui/assets/35179038/d7d0dc17-dbd8-4376-99c8-eb8e90b1c937">